.github/workflows/python-flake8.yml

```diff
@@ -31,4 +31,4 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install flake8
-        flake8 --count --max-line-length=127 --statistics mslib tests
+        flake8 --count --statistics mslib tests
```

Comment:
we don't run it on docs, see parameters.

Reply:
when you fixed docs now, then extend the parameters.

Reply:
What do i do now then?

Reply:
you did a bit too much, as in the "good first" issue was defined, read line 34, extend it to the docs dir too.

Reply:
ok understood. actually its my first tym doing a contribution that way i got confused.
so here is the thing i think i should do now say yes and if i understood the assignment correctly.
i will fix all the issue in #2111 but this time from the stable branch.
and i will leave the python-flake8 file as it is
?

Reply:
don't touch the conf.py - no changes. Only remove the extra parameter from the CI file.

Look also in my comment in the slack channel

Reply:
the reason for _good first issue_, is to get a simple thing done, by learning the workflow. There bigger other issues later.